roles/repositories/tasks/main.yml

```diff
@@ -1,33 +1,7 @@
-- name: Load Repos Files
-  ansible.builtin.include_vars:
-    file: "{{ item }}"
-    hash_behaviour: "merge"
-  loop: "{{ lookup('fileglob', ros_repos_directory + '*.repos', wantlist=True) }}"
+- name: Install Repositories - From File
+  ansible.builtin.include_tasks: from_file.yml
+  when: read_repositories_from_file
```

Comment:
The flag `read_repositories_from_file` might be removed. Check if the file list is not empty instead.

Reply:
I thought before that loading implicit meanings to some variables is not the best solution. For that suggested case defining a variable and overloading variable `repositories_file_name`, the file list, will do that. 
Two choices arise:
- Less number of variables, implicit meanings on variables. Less number of variables may lower the changes of small bugs.
- More number of vars, no implicit meaning, everything will be set separately. A little bit easier to read, but a little bit likely to have small bugs due to the wrong declaration.
Should we still change?

Reply:
We always want to separate different things under different names. (low coupling)
We always want to collect similar things under the same/similar name. (high cohesion)

There is no blind policy to minimize or maximize the number of things (here, variables).

Also recall: https://en.wikipedia.org/wiki/Coupling_(computer_programming)

Reply:
OK, got it. I will change it now.